src/modules/launcher/Plugins/Microsoft.PowerToys.Run.Plugin.System/plugin.json

```diff
@@ -2,6 +2,7 @@
   "ID": "CEA08895D2544B019B2E9C5009600DF4",
   "ActionKeyword": "",
   "IsGlobal": true,
+  "ConfirmSys": false,
```

Comment:
This option should not be here. Take a look at how it is implemented for the Windows Search plugin https://github.com/microsoft/PowerToys/blob/91797669ee0c6536a2ea6b0809411e4db0a284b5/src/modules/launcher/Plugins/Microsoft.Plugin.Indexer/Main.cs#L54-L62

![image](https://user-images.githubusercontent.com/17161067/126307030-584348ef-e233-43b3-a376-2819c022836e.png)

Reply:
Done

Reply:
The checkbox should be displayed only under the Windows System Commands plugin